other/files/svn-exposed.bcheck

```diff
@@ -0,0 +1,32 @@
+metadata:
+    language: v1-beta
+    name: "SVN Configuration File Exposed"
+    description: "Tests for exposed svn config in current path and at the root directory of site"
+    author: "j3ssie"
+    tags: "exposure", "svn", "config", "file"
+
+run for each:
+    potential_path = ".svn/entries", ".svn/text", ".svn/all-wcprops", "CVS/Entries"
+
+given request then
+    # replace the potential path with the last path
+    send request called check:
+        method: "GET"
+        replacing path: `{regex_replace({regex_replace({base.request.url}, "^.*?\/.*?\/.*?\/", "/")}, "([^/]+)$", "")}{potential_path}`
+
+    # replace the potential path with entire URI
+    send request called check1:
+        method: "GET"
+        replacing path: `{regex_replace({base.request.url}, "^.*", "")}/{potential_path}`
+
+    if {latest.response.status_code} is "200" then
```

Comment:
Would it be clearer to reference check1 here rather than latest?

Reply:
I use the `{latest.response}` is for the purpose above https://github.com/PortSwigger/BChecks/pull/175#discussion_r1481863110 to check for both of the requests. Do you think it would be better to do 2 separate if statements for `check` and `check1` ?